.github/workflows/pytest.yaml

```diff
@@ -17,8 +17,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
@jenshnielsen, please ignore my ignorance: Does this simply mean we want to run the tests on the last four python versions on linux and only on the latest on Windows?

Reply:
Yes, this is rather random and I think we should do a different pr to sub select the most relevant elements from the maxtrix. For now I just ported the one version on windows from conda to here. 

Exclude only seems to take since values (no arrays) so each job has to be excluded separately. There is a trick where you first exclude all jobs and then reinclude the one that you want but this seems more readable (if more verbose)